src/bot.py

```diff
@@ -35,8 +35,11 @@
     ]
 )
 
+mininterval = config['mininterval']
+maxinterval = config['maxinterval']
+hours_to_wait = randint(mininterval, maxinterval) * 24

```

Comment:
Additionally, a comment explaining what I just said in a very condensed manner would be pretty decent too. Thanks!